doc/api/test.md

```diff
@@ -678,11 +678,10 @@ added:
   properties are supported:
   * `concurrency` {number|boolean} If a number is provided,
     then that many files would run in parallel.
-    If truthy, it would run (number of cpu cores - 1)
-    files in parallel.
-    If falsy, it would only run one file at a time.
+    If `true`, it would run `os.availableParallelism() - 1` files in parallel.
+    If `false`, it would only run one test at a time.
     If unspecified, subtests inherit this value from their parent.
```

Comment:
AFAIK it can never be a subtest.
```suggestion
```

Reply:
The way that I read this is: when the option isn't passed to `test`, any `t.test` within `test` will inherit that value. 

The same text is also used in the `options.timeout` description and a few other places in this doc.